MediaManager/Plugin.MediaManager.Abstractions/Implementations/MediaManagerBase.cs

```diff
@@ -72,7 +72,7 @@ private IPlaybackManager CurrentPlaybackManager
 
         public event MediaFileFailedEventHandler MediaFileFailed;
 
-        private IMediaFile _currentMediaFile => MediaQueue.Current;
+        private IMediaFile CurrentMediaFile => MediaQueue.Current;
```

Comment:
No, that's only for properties as far as I know. My IDE suggested the change to be according to the naming conventions, so I think it should be that way.